includes/commands/class-mu-migration-posts.php

```diff
@@ -108,19 +109,29 @@ function ( $result ) use ( &$equals_id, &$author_not_found, $ids_map, $verbose,
 					$author_not_found[] = absint( $result->ID );
 				}
 
+				// Automatically add Woocommerce user id field
 				if ( $is_woocommerce ) {
-					$old_customer_user = get_post_meta( (int) $result->ID, '_customer_user', true );
-
-					if ( isset( $ids_map->{$old_customer_user} ) && $old_customer_user != $ids_map->{$old_customer_user} ) {
-						$new_customer_user = $ids_map->{$old_customer_user};
-
-						update_post_meta( (int) $result->ID, '_customer_user', $new_customer_user );
-
-						$this->log( sprintf(
-							__( 'Updated customer_user for "%s" (ID #%d)', 'mu-migration' ),
-							$result->post_title,
-							absint( $result->ID )
-						), $verbose );
+					$this->assoc_args['uid_fields'] .= ',_customer_user';
```

Comment:
To make the code more readable first explode `uid_fields` . That will create an array (an empty one if no arg is passed), then you add `_customer_user` to that array and run it through array_filter.

```
$uid_fields = explode( ',', $this->assoc_args['uid_fields'] );
if ( $is_woocommerce ) {
   $uid_fields[] = '_customer_user';
}
$uid_fields = array_filter( $uid_fields );
```